runtime/compiler/compile/J9Compilation.cpp

```diff
@@ -183,21 +185,9 @@ J9::Compilation::Compilation(int32_t id,
 #endif /* defined(J9VM_OPT_JITSERVER) */
    _osrProhibitedOverRangeOfTrees(false)
    {
-#if defined(J9VM_OPT_JITSERVER)
-   // In JITServer, we would like to use JITClient's processor info for the compilation
-   // The following code effectively replaces the cpu with client's cpu through the getProcessorDescription() that has JITServer support
-   if (self()->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER)
+   if (((TR_J9VMBase *)fe)->isAOT_DEPRECATED_DO_NOT_USE() && TR::Compiler->target.cpu.isX86())
```

Comment:
```cpp
if (((TR_J9VMBase *)fe)->isAOT_DEPRECATED_DO_NOT_USE() && TR::Compiler->target.cpu.isX86())
      _target = TR::Compiler->relocatableTarget;
```

I think this needs to be moved out as well (It looks like you already added code outside the constructor but didn't remove it from here)

Reply:
Sorry forgot about this! Fixed in e7c02f3.